_data-prepper/pipelines/configuration/processors/csv.md

```diff
@@ -10,21 +10,27 @@ nav_order: 45
 
 ## Overview
 
-Takes in an Event and parses its CSV data into columns.
+The `csv` processor takes in an event and parses its comma-separated values (CSV) data into columns.
```

Comment:
It is probably not accurate to say "ingests". Perhaps it could simply say:

> The `csv` processor parses event data as comma-separated values (CSV).